### PR TITLE
w32tm: add page

### PR DESCRIPTION
**Pull request by FallenGameR**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** versions implicitly follow Windows OS version, but these commands should work in any windows from WS2012 up until now

### CODE REVIEW

File: pages/windows/w32tm.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show a time offset graph against a time server:
+
+`w32tm /stripchart /computer:{{time_server}}`
+
+- Show an NTP reply from a time server:
+
+`w32tm /stripchart /packetinfo /samples:1 /computer:{{time_server}}`
+
+- Show state of the currently used time servers:
```
Comment (kbdharun) [inline pages/windows/w32tm.md:18]:
```suggestion
- Show the state of the currently used time servers:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have a minor suggestion for this page.

File: pages/windows/w32tm.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# w32tm
+
+> Query and control w32time time synchronization service on Windows from the command-line.
```
Comment (acuteenvy) [inline pages/windows/w32tm.md:3]:
```suggestion
> Query and control the w32time time synchronization service.
```

File: pages/windows/w32tm.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://learn.microsoft.com/en-us/windows-server/networking/windows-time-service/windows-time-service-tools-and-settings>.
+
+- Show the current status of the time synchronization:
```
Comment (acuteenvy) [inline pages/windows/w32tm.md:6]:
```suggestion
- Show the current status of time synchronization:
```

**Review by acuteenvy — APPROVED** [review]

**Comment by FallenGameR** [comment]
@kbdharun, I've addressed the comment you had. Is PR ok now?

**Review by kbdharun — APPROVED** [review]

**Pull request merged**